.php_cs.dist

```diff
@@ -37,6 +37,7 @@ return PhpCsFixer\Config::create()
         'semicolon_after_instruction' => true,
         'strict_comparison' => true,
         'strict_param' => true,
+        'method_chaining_indentation' => true,
```

Comment:
Your last commit moved this rule back to the end of the list. Please move it after `list_syntax` again.